drivers/SmartThings/zigbee-humidity-sensor/src/aqara/init.lua

```diff
@@ -0,0 +1,75 @@
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+local clusters = require "st.zigbee.zcl.clusters"
+local capabilities = require "st.capabilities"
+
+local TemperatureMeasurement = clusters.TemperatureMeasurement
+local RelativeHumidity = clusters.RelativeHumidity
+local PowerConfiguration = clusters.PowerConfiguration
+
+local FINGERPRINTS = {
+  { mfr = "LUMI", model = "lumi.sensor_ht.agl02" }
+}
+
+local configuration = {
+  {
+    cluster = TemperatureMeasurement.ID,
+    attribute = TemperatureMeasurement.attributes.MeasuredValue.ID,
+    minimum_interval = 1,
+    maximum_interval = 3600,
+    data_type = TemperatureMeasurement.attributes.MeasuredValue.base_type,
+    reportable_change = 1
+  },
+  {
+    cluster = RelativeHumidity.ID,
+    attribute = RelativeHumidity.attributes.MeasuredValue.ID,
+    minimum_interval = 1,
+    maximum_interval = 3600,
+    data_type = RelativeHumidity.attributes.MeasuredValue.base_type,
+    reportable_change = 1
+  },
+  {
+    cluster = PowerConfiguration.ID,
+    attribute = PowerConfiguration.attributes.BatteryVoltage.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
+    data_type = PowerConfiguration.attributes.BatteryVoltage.base_type,
+    reportable_change = 1
+  }
```

Comment:
The default attribute configurations are very similar to what you have here (30s minimum interval for temp/humidity and a longer max interval for battery voltage). 

If you used the defaults here this could be a very small file that just changes the init to use battery voltage. Are those changes that valuable to you? They seem like they'd have some non-negligible effect on reducing the battery life of the device.

Reply:
@sbkimaqara 
As Green's comment, This is the default min/max configuration of each cluster
- TemperatureMeasurement : min/max(30,300)
- RelativeHumidityMeasurement: min/max(30,3600)
- PowerConfiguration: min/max(30, 21600)
He recommended to use default min of temp/humidity, max of power for battery consumption.

Reply:
What do you think about below?
- TemperatureMeasurement : Updated
- RelativeHumidityMeasurement : Updated
- PowerConfiguration : Kept at 30/3600 which is the same as the Configuration Interval of Contact Sensor.